etl/example.rkt

```diff
@@ -0,0 +1,16 @@
+#lang racket
+
+(provide (contract-out
+          [etl (-> hash? hash?)]))          
+
+; TODO Provide contract on the keys and values of the input and output
```

Comment:
Probably `(hash/c exact-nonnegative-integer? (listof string?)) ...`, but it'd be cool to make contracts for lowercase & uppercase letters.